samples/nrf9160/nrf_cloud_rest_device_message/Kconfig

```diff
@@ -29,6 +29,13 @@ config REST_DEVICE_MESSAGE_DO_JITP
 	  nRF Cloud, enable this option and press the configured button when
 	  prompted after the sample starts.
 
+config REST_DEVICE_MESSAGE_TEST_BUILD
```

Comment:
can this be changed to something-RTT instead of "TEST_BUILD"

Reply:
Will do that and also move `date_time` related configs to `prj.conf` due to failure in twister build below.